modules/appeals_api/app/serializers/appeals_api/higher_level_review_serializer.rb

```diff
@@ -3,7 +3,7 @@
 class AppealsApi::HigherLevelReviewSerializer
   include FastJsonapi::ObjectSerializer
   set_key_transform :camel_lower
-  attributes :status, :updated_at, :created_at
+  attributes :status, :code, :detail, :updated_at, :created_at
```

Comment:
Let's make sure to only show `code` and `detail` if the status is `error`. The rest of the time they can be hidden - sorry I forgot to add that detail into the ticket :(

Reply:
We decided NOT to return code and detail, even for appeals with an error status......not much a caller could do with the error